hack/util.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```

Comment:
Not a fan of utilities; they tend to grow with unrelated stuff making the file hard to read/maintain. 

In case we really need the fancy messages, I recommend moving the styling to its own file or to common.sh

Reply:
I created util.sh by pulling out some functions from errata-check.sh to re-use in merge-order.sh. I can move all this to common.sh if that's the consensus